clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/KubernetesUtil.groovy

```diff
@@ -112,4 +85,29 @@ class KubernetesUtil {
     return securityGroups
   }
 
+  static String securityGroupKey(String securityGroup) {
+    return String.format("$SECURITY_GROUP_LABEL_PREFIX%s", securityGroup)
```

Comment:
Since `return` is optional in groovy, should we drop it here? @duftler @ttomsu what have you guys been doing?